keyboards/lets_split/keymaps/DE_programming/keymap.c

```diff
@@ -0,0 +1,151 @@
+#include QMK_KEYBOARD_H
+#include "keymap_german.h"
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTZ 0
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```

Comment:
```suggestion
```

It’s usually a bad idea to skip a bunch of layers.  It blows up firmware size by creating “phantom” layers that aren’t explicitly defined.  Another supporting argument for `enum`.